apps/client/src/common/api/viewSettings.ts

```diff
@@ -1,21 +1,18 @@
 import axios from 'axios';
-import { ViewSettings } from 'ontime-types';
+import type { ViewSettings } from 'ontime-types';
 
-import { apiEntryUrl } from './constants';
+import { ontimeQueryClient } from '../../common/queryClient';
 
-const viewSettingsPath = `${apiEntryUrl}/view-settings`;
+import { apiEntryUrl, VIEW_SETTINGS } from './constants';
+const viewSettingsPath = apiEntryUrl + '/view-settings';
 
-/**
- * HTTP request to retrieve view settings
- */
-export async function getView(): Promise<ViewSettings> {
+export async function getViewSettings() {
   const res = await axios.get(viewSettingsPath);
   return res.data;
 }
 
-/**
- * HTTP request to mutate view settings
- */
 export async function postViewSettings(data: ViewSettings) {
-  return axios.post(viewSettingsPath, data);
+  await ontimeQueryClient.cancelQueries({ queryKey: VIEW_SETTINGS });
```

Comment:
cleanup: this shouldnt be here. we dont want to mix up axios and react-query
this logic can move to the onMutate callback of the useMutation, like

```ts
useMutation({
    mutationFn: postViewSettings,
    onMutate: async () => {
        await ontimeQueryClient.cancelQueries({ queryKey: VIEW_SETTINGS });
    },
    onSuccess: (data) => {
      ontimeQueryClient.setQueryData(VIEW_SETTINGS, data);
    },
})
```